product/views.py

```diff
@@ -0,0 +1,277 @@
+import json
+from datetime import datetime
+from random   import randint
+
+from django.http  import JsonResponse, HttpResponse
+from django.views import View
+
+from user.utils     import login_decorator
+from product.models import (Menu,
+                            Category, 
+                            Product,
+                            Image,
+                            ProductStock,
+                            RelatedProduct,
+                            Disease,
+                            ProductDisease,
+                            Allergy,
+                            ProductAllergy,
+                            Goal,
+                            ProductGoal,
+                            ProductDietaryHabit,
+                            DietaryHabit,
+                            VeganLevel,
+                            ActivityLevel,
+                            GenderCode,
+                            AgeLevel)
```

Comment:
이렇게 너무 길어질 경우 아래와 같이 정렬해주실 수 있겠네요~!
```suggestion
from product.models import (
        Menu, Category, Product,
        Image, ProductStock, RelatedProduct,
        Disease, ProductDisease, Allergy,
        ProductAllergy, Goal, ProductGoal,
        ProductDietaryHabit, DietaryHabit, VeganLevel,
        ActivityLevel, GenderCode, AgeLevel
)
```

Reply:
수정했습니다!